.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,15 +1,6 @@
 <!--
 Thank you for submitting to the TechEmpower Framework Benchmarks!
 
-*** PLEASE READ ***
-
-We are transitioning the testing suite to use docker. If you are opening a pull request to update a framework, please check the docker branch first to make sure the test hasn't already been converted. If you notice the test has a dockerfile, the pull request should be made against the docker branch. 
-
-Any new framework tests should be made against the docker branch. Round 16 and beyond will use this new setup. As it will take some time to update the documentation to reflect these changes, feel free to ping any of us for guidance.
```

Comment:
The documentation still isn't up-to-date, so I guess a sentence about that should stay.